### PR TITLE
[Build-System] freeswitch.spec update

### PR DESCRIPTION
**Pull request by sergey-safarov**
Please check to commit comments for details.

### CODE REVIEW

File: freeswitch.spec [hunk 1]
```diff
@@ -122,12 +122,10 @@ Vendor:       	http://www.freeswitch.org/
 #
 ######################################################################################################################
 Source0:        http://files.freeswitch.org/%{name}-%{nonparsedversion}.tar.bz2
-Source1:	http://files.freeswitch.org/downloads/libs/v8-3.24.14.tar.bz2
-Source2:	http://files.freeswitch.org/downloads/libs/mongo-c-driver-1.1.0.tar.gz
```
Comment (s3rj1k) [inline freeswitch.spec:126]: @sergey-safarov  Source2 is missing (Source0, Source1, Source3, ...)

**Review by s3rj1k — CHANGES_REQUESTED** [review]

File: freeswitch.spec (hunk 1 above)
Comment (sergey-safarov) [reply freeswitch.spec:126]: That is not important.
In future, we can use Soruce2 for another file.

Comment (s3rj1k) [reply freeswitch.spec:126]: It looks bad, please move Source5 to Source2

Comment (sergey-safarov) [reply freeswitch.spec:126]: PR updated

Comment (s3rj1k) [reply freeswitch.spec:126]: @sergey-safarov Thanks

**Review by s3rj1k — APPROVED** [review]
LGTM

**Comment by sergey-safarov** [comment]
Any update on this?

@andywolk, will you finish the review?

**Review by andywolk — APPROVED** [review]

**Comment by sergey-safarov** [comment]
Tanks @andywolk 
What is next speps?

**Comment by andywolk** [comment]
@sergey-safarov I still need to run one last internal test with this change. Planning to do that this week.

**Comment by andywolk** [comment]
@sergey-safarov Finished my checks. Doing some preparations in our infrastructure before merging this.

**Pull request merged**